src/configuration/configuration.c

```diff
@@ -0,0 +1,186 @@
+#include <ev.h>
```

Comment:
Could we not just have this be a regular UNIX-style program that processes the file and then exits? Then we run that as a task in `/etc/inittab` (I suppose there is no time to move to a NetBox base using finit?) before starting dnsmasq as a regular service.

We could even replace the program with a direct call to `sed`.

Reply:
+1 that would be great.

Reply:
Changed to run a script that calls sed from /etc/inittab.